ember.md

```diff
@@ -94,15 +94,14 @@ get(someObject, 'isUpdated'); //true
 <a name="computed-properties--brace-expansion"></a>
 ### 3.1 Brace Expansion
 
-When a computed property depends on multiple properties, we can declare the dependents using brace expansion. It is useful for cases when the computed property depends on multiple properties of an object. Using brace expansion in computed properties makes our code more organized and easier to read, as it organizes dependent keys. 
+> Why? When a computed property depends on multiple properties, always use the dependents using brace expansion. It is useful for cases when the computed property depends on multiple properties of an object. Using brace expansion in computed properties makes our code more organized and easier to read, as it organizes dependent keys. 
```

Comment:
This description is still unclear to me. "always use the dependents using brace expansion" doesn't make sense.
Maybe something like "always declare dependents of the same object using brace expansion"?
@ASH-Anthony @ASH-Bryan Thoughts?

Reply:
How about: "When a computed property depends on multiple properties of the same object, specify the properties using brace expansion."